compiler/repl-service/server/src/main/scala/com/digitalasset/daml/lf/speedy/ReplServiceMain.scala

```diff
@@ -278,7 +278,7 @@ class ReplService(
               }
             case RunScriptRequest.Format.JSON =>
               try {
-                LfValueCodec.apiValueToJsValue(v.toValue).compactPrint
+                LfValueCodec.apiValueToJsValue(v.toUnNormalizedValue).compactPrint
```

Comment:
```suggestion
                LfValueCodec.apiValueToJsValue(v.toUnnormalizedValue).compactPrint
```
nit: just looks weird to me otherwise